approaches/byte-pair-encoding.md

```diff
@@ -5,3 +5,25 @@ layout: coming_soon
 title: Byte-pair encoding
 description: A data compression algorithm for machine translation
 ---
+
+Byte-pair encoding (BPE) is a data compression algorithm where the most common consecutive pair of bytes is replaced by a byte that is not seen in the data.
+It was first introduced in the article titled "A new Algorithm for Data Compression" by Philip Gage in the February 1994 edition of C Users Journal.
```

Comment:
According to the [style guide](https://machinetranslate.org/style#apostrophes-and-quotation-marks), curly quotation marks should be used.

Reply:
Maybe for this we should use Italics?